apps/frontend-pwa/src/components/liveSession/FeedbackArea.tsx

```diff
@@ -80,7 +80,15 @@ function Subscriber({ subscribeToMore, sessionId }) {
   return <div></div>
 }
 
```

Comment:
On the student page, there is still a bug in the state logic, which could possibly also be fixed with this PR.

If the window size is decreased to mobile, the user switches to the feedback view and then increases the window size again, the question does not appear in a side-by-side view as it should.

Reply:
fixed ✅

Reply:
Great, thanks! this has been a bug for a long time already! 😄